### DISCUSSION TITLE
What is the intended way to insert and general image?

### DISCUSSION THREAD

**Discussion by neztEx** [twilio-labs/paste#3489] [General] [answered]
I see the Media Object component and Box as ways of inserting an image, is there a specific component that handles that or what is Paste intended design on this?

**Comment by SiTaggart** [comment]
Hi @neztEx 

Great question! 

Right now, we don't have any preferred way because our intention is to work with any React based Front End Framework. Each one of those has their own way of doing images, which sometimes includes some really great optimization techniques.

Because of that, we can't provide a single, one size fits all solution to images. It would be foolish of us to try with so many great solutions out there already.

If you're using Next you should use Next Image, Gatsby, use Gatsby Image. Plain old static assets, just use an HTML `<img />` tag. 

If you want to hook into Styling Props you can render `<Box as="img" src="path/to/image/asset" />`, it's just HTML underneath.

Our advice is to render images how your app or framework renders images, as they probably do it better than we would.

**Reply by neztEx** [reply, marked as answer]
Awesome, thanks!
